src/domain/synthetics/markets/utils.ts

```diff
@@ -162,6 +176,31 @@ export function getAvailableUsdLiquidityForCollateral(marketInfo: MarketInfo, is
   return liqudiity;
 }
 
+export function getMarketReservesAccountingInterest(marketInfo: MarketInfo, isLong: boolean) {
+  if (marketInfo.isSpotOnly) {
+    return {
+      reserveUsd: BigNumber.from(0),
+      maxReserveUsd: BigNumber.from(0),
+    };
+  }
+
+  const reservedUsd = getReservedUsd(marketInfo, isLong);
+  const maxReservedUsd = getMaxReservedUsd(marketInfo, isLong);
+
+  const openInterestUsd = getOpenInterestUsd(marketInfo, isLong);
+  const maxOpenInterestUsd = getMaxOpenInterestUsd(marketInfo, isLong);
+
+  const [currentMaxReserve, currentReserve] = getArrayWithSmallerDifference(
+    [maxReservedUsd, reservedUsd],
+    [maxOpenInterestUsd, openInterestUsd]
+  );
+
+  return {
+    reserveUsd: currentReserve,
```

Comment:
i think name "reserve" can be confusing since it's either "reserve" or "open interest" in reality